src/calculation/SaleElement.ts

```diff
@@ -0,0 +1,28 @@
+import { Currency } from '~/models/Currency';
+import { Sale } from '~/models/Sale';
+import { VatRate } from '~/models/VatRate';
+
+import { roundSum } from './roundSum';
+
+export class SaleElement implements Sale {
+  public code: string | null = null;
```

Comment:
I think `pkw` (or rather `pkd`? :smiley:) is rather specific to Polish market. If someone else reads the code then `code` is probably easier to understand. But I think it can still be renamed - I propose `transactionCode` or `classificationCode`, what to you think?

Reply:
Decision: we use `classificationCode`.